lib/quby/answers/services/updates_answers.rb

```diff
@@ -29,14 +28,19 @@ def update(new_attributes = {})
             else
               started_at = nil
             end
-
             answer.mark_completed(started_at)
             answer.outcome = OutcomeCalculation.new(answer).calculate
             Quby.answers.update!(answer)
             succeed!
           else
             fail!
           end
+        rescue
+          # we dont save answer since it probably has a weird state now, instead we save the a clone of answer
+          # that just has the raw_params set on it, so we can do data recovery with the raw_params
+          backup_answer.raw_params['could_not_update_at'] = DateTime.now.to_i # to note this is was a failed update
+          Quby.answers.update!(backup_answer)
```

Comment:
Goed voorstel, echter is de answerrepo en het answer zelf beide niet een activerecord. Eeen refactor zodat je validaties kan overslaan is wat lastig en valt wat mij betreft buiten het ticket. Ik vermoed ook dat er weinig validaties zouden kunnen falen aangezien alle answer validaties juist binnen deze use case vallen en QubyAnswerRepo::Record in RoQua op dit moment geen validaties bevat.